bin/update

```diff
@@ -28,18 +40,20 @@ chdir APP_ROOT do
   # If we need a "reset all the things" script, we should call it bin/reset.
   system! 'bundle update'
 
+  # load 'Rakefile'
+  load_rakefile
   puts "\n== Updating database =="
-  system! 'bin/rails db:migrate'
+  Rake::Task["db:migrate"].invoke
 
   puts "\n== Seeding database =="
-  system! "bin/rails db:seed GOOD_MIGRATIONS=skip"
+  Rake::Task["db:seed"].invoke
```

Comment:
As an FYI, if we were to combine args into a single would be doing the same thing I am doing here:

https://github.com/ruby/rake/blob/master/lib/rake/application.rb#L149-L153

Coming from `Rake::Application#top_level` <= `Rake::Application#run` <= `bin/rake`.  The only difference is between what I am doing and something like:

```
`bin/rake db:migrate db:seed`
```

Is the system overhead in making a new process (while I am far from an expert in the subject, this should be noticeable in the difference in the `sys` metrics between the two).  Both will only load the env once if other prereq tasks have been loaded already, and will be called in the same fashion, using `invoke`.  Without the autoloading that we do, this would be compounded further.